packages/examples/src/animation/center-shrink-circle.sub

```diff
@@ -1,2 +1,2 @@
-Object o 
+Object o
```

Comment:
This file change is causing a merge conflict because of #1280; could you either remove this edit or pull from `main` and then push to your branch again with the new `.substance` file extension? (I would do it myself but your branch is on your fork instead of in this source repo 😛)